pkg/assembler/graphql/schema/vulMetadata.graphql

```diff
@@ -0,0 +1,120 @@
+#
+# Copyright 2023 The GUAC Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# NOTE: This is experimental and might change in the future!
+
+# Defines a GraphQL schema for the Vulnerability Metadata
+
+"Records the type of the score being captured by the score node"
+enum VulnerabilityScoreType {
+  CVSSv2
+  CVSSv3
+  EPSSv1
+  EPSSv2
+}
+
+"""
+VulnerabilityScore node captures the score type and its corresponding value. There is a
+one-to-one relationship between the VulnerabilityMetadata and the score.
+
+Examples:
+
+type: EPSSv1
+value: "0.960760000"
+
+type: CVSSv2
+value: "5.0"
+
+type: CVSSv3
+value: "7.5"
+"""
+type VulnerabilityScore {
+  id: ID!
+  type: VulnerabilityScoreType!
+  value: String!
```

Comment:
hmm thoughts about this being a float? would be easier to do comparisons.

Reply:
Would a future "score" be something like "high, medium, low" ?

Reply:
Right based on that if it should be string...?

Reply:
Well thinking about this... that would be criticality that we will not be capturing. That will be determined based on the type and score

Reply:
yea imo i think this is a place where we can be opiniated because it adds utility. anything that doesn't fall under can be put under criticality or any other generic metadata

Reply:
Could this be CVSS or similar?

Reply:
yes this is for CVSS, EPSS etc...

Reply:
changed.

Reply:
In that case, it should be string, I think